pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[tool.poetry]
+name = "qctrl-pyquil"
+version = "0.0.1"
+description = "Q-CTRL Pyquil Adapter"
+license = "Apache-2.0"
+authors = ["Q-CTRL <support@q-ctrl.com>"]
+readme = "README.md"
+keywords = [
+    "quantum", 
+    "computing", 
+    "open source", 
+    "engineering",
+    "pyquil"
+]
+classifiers = [
+            "Development Status :: 5 - Production/Stable",
+            "Environment :: Console",
+            "Intended Audience :: Developers",
+            "Intended Audience :: Education",
+            "Intended Audience :: Science/Research",
+            "License :: OSI Approved :: Apache Software License",
+            "Natural Language :: English",
+            "Operating System :: OS Independent",
+            "Programming Language :: Python :: 3.6",
+            "Topic :: Scientific/Engineering :: Physics",
+            "Topic :: Scientific/Engineering :: Visualization",
+            "Topic :: Software Development :: Embedded Systems",
+            "Topic :: System :: Distributed Computing"
+        ]
+repository = "https://github.com/qctrl/python-pyquil"
+packages = [
+    { include = "qctrlpyquil" },
+]
+
+[tool.poetry.dependencies]  # https://poetry.eustace.io/docs/versions
+python = ">=3.6.4,<3.8"
+numpy = "^1.16"
+scipy = "^1.3"
+toml = "^0.10.0"
+pyquil = "^2.9"
+qctrl-open-controls = "^3.0.0"
```

Comment:
Why does this require qctrl-open-controls?

Reply:
Because the base classes such as `sequence` and `controls` are defined in `open-controls`; so it takes object of that form and converts those into say `pyquil` program.